components/content/Team.jsx

```diff
@@ -77,7 +77,7 @@ class TeamContent extends React.Component {
   onClose = () => {
```

Comment:
It would be great if we could remove all the duplicate onClose functions and have them share one method that takes props, and settings perhaps.